doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -699,6 +699,83 @@ If you need to remove the Tarantool Kubernetes operator, execute:
       persistentvolumeclaim "www-routers-1-0" deleted
       persistentvolumeclaim "www-storages-0-0" deleted
 
+
+.. _cartridge_kubernetes_cluster_management_failover:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Failover
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Failover - replicaset leader selection mechanism. You can read more about it `here <https://www.tarantool.io/en/doc/latest/book/cartridge/topics/failover/>`_.
```

Comment:
There's no such document. https://www.tarantool.io/en/doc/latest/book/cartridge/topics/failover

If you want to make an internal link, use the [`:doc:` or `:ref:` link syntax](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/#linking-to-document).

Reply:
I want to save working links in github guide version.